### PR TITLE
Arthur dev

### PR DESCRIPTION
**Pull request by arthuryoung-cs107**
Merging all architecture stuff

### CODE REVIEW

File: src/LUNA_architecture_tests/LUNA_architecture_check_series1.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+import pandas as pd
+import time
+from IPython.display import display
+
+from src.models import LUNA
+from src.utils import generate_data
+import LUNA_database_parse as par
+
+def run_experiments():
+    x, y, _ = generate_data(number_of_points=50, noise_variance=9)
```
Comment (yoonspark) [inline src/LUNA_architecture_tests/LUNA_architecture_check_series1.py]: Due to the change in function definition, it has to be: `x, y, _, _ = generate_data...`

Comment (arthuryoung-cs107) [reply src/LUNA_architecture_tests/LUNA_architecture_check_series1.py]: oh u right lemme fix

Comment (arthuryoung-cs107) [reply src/LUNA_architecture_tests/LUNA_architecture_check_series1.py]: thats odd I was able to run it

Comment (yoonspark) [reply src/LUNA_architecture_tests/LUNA_architecture_check_series1.py]: It's perhaps because your version of `utils.py` is not up to date.

**Review by yoonspark — APPROVED** [review]

**Pull request merged**